code/modules/species/species.dm

```diff
@@ -104,7 +104,7 @@
 	var/gibbed_anim =   "gibbed-h"
 	var/dusted_anim =   "dust-h"
 
-	var/death_sound = 'sound/effects/deathrattle.ogg'
+	var/death_sound
```

Comment:
Don't just keep it sitting there like that unassigned, it's still used in `code/modules/mob/living/carbon/human/death.dm` at Line 73 / 74. Remove that too if you are gonna completely remove this, don't leave an unused var there like that.

Reply:
If you are gonna remove it, then remove it, and double check that it's not used by anything.